src/mbgl/gl/drawable_gl.cpp

```diff
@@ -23,7 +23,7 @@ DrawableGL::~DrawableGL() {
 }
 
 void DrawableGL::draw(PaintParameters& parameters) const {
-    MLN_TRACE_FUNC();
+    MLN_TRACE_FUNC()
```

Comment:
I thought we were doing the opposite. It doesn't feel right to have instructions without the `;` at the end. What if we change the macros to actual functions? Think of the `assert` macro in `assert.h`

Reply:
Also clang-format doesn't interpret macros as it's tidying the actual code not pre-processed one

Reply:
I'm not sure if it's coming from clang, but if the semicolons are outside the macro, I get a ton of these in AndroidStudio:

![image](https://github.com/user-attachments/assets/c49dd521-7ffe-4368-b3b3-ee57e6989fe2)

Reply:
we need to use `(void)(0)` when Tracy is disabled to avoid the empty statement spam

Reply:
Already used that.

Reply:
@TimSylvester What platform?

Reply:
That was Android Studio on MacOS, but while it was evaluating to nothing, so I expect the above will resolve it.

Reply:
Seems to work OK now.